src/dialog.tsx

```diff
@@ -124,26 +124,35 @@ interface BaseDialogProps {
     title?: string;
```

Comment:
title in alert & confirm dialog have different text-preset than Dialog
![image](https://user-images.githubusercontent.com/6722153/195021978-6eb3045d-77f1-43b0-9a34-66446d12ec3a.png)

Reply:
why?

Reply:
hmmm... that argument sounds weak to me. I'd say it's way better to have consistence across dialogs in the same platform (web) than having consistence between iOS, Android and web

Reply:
I think is better to break the consistency between this type of dialogs because the usage is very different. 

Confirm and alerts it's for immediate actions, however, dialogs could have much plane information without a required action. The moment for the user is very different.